### PR TITLE
Fixed #645: Restored the section

### LINKED ISSUE #645: Thank you for contributing
The section has disappeared

### PR DESCRIPTION
**Pull request by samyakbvs**
The Thanks section had disappeared I added it

### CODE REVIEW

**Comment by ParthS007** [comment]
@samyakbvs  Please provide a preview link 👍

File: index.html [hunk 1]
```diff
@@ -132,8 +132,7 @@ <h2 class="animated fadeIn">Contributors</h2>
 <section class="bg-dark">
 	<div class="container">
 		<div class="col-md-10 col-md-offset-1 col-sm-7 col-sm-offset-1 v-align-transform text-center">
-			<h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing!", "योगदान करने के लिए धन्यवाद!", "Danke für den Beitrag!", "Dziękujemy za Twój wkład!", "貢献していただきありがとうございます!", "பங்களித்ததற்கு நன்றி!", "Tak for bidrag!", "Grazie per aver contribuito!", "Go raibh maith agat as ranníoc!",  "Merci de contribuer!", "공헌 해 주셔서 감사합니다!", "অবদান রাখার জন্য আপনাকে ধন্যবাদ!", "දායක වූවාට ස්තුතියි!", "Gracias por contribuir!", "Спасибо Вам за Ваш вклад!", "Σας ευχαριστούμε για τη συμβολή!", "ขอบคุณสำหรับการมีส่วนร่วมกับเรา!", "Bedankt voor de bijdrage!", "
```
Comment (0x48piraj) [inline index.html:135]: You just deleted it !

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
I guess, a Malformed PR

**Comment by samyakbvs** [comment]
It is not working, see the fossasia site

**Comment by samyakbvs** [comment]
Just wait for sometime I will fix this issue

**Pull request closed without merging**